cmd/operator-ui/actions/hubs.go

```diff
@@ -88,6 +88,7 @@ func (v HubsResource) New(c buffalo.Context) error {
 			Size: "150Gi",
 		},
 	}
+	blackduck.Spec.ExternalPostgres = &v2.PostgresExternalDBConfig{}
```

Comment:
Creating an empty config here would prevent the later checks against nil from succeeding wouldn't it?  You would only want to create this struct if there is an external db config I would think.

Reply:
Good catch. The web framework requires that we initialise the struct so I just updated the Create function and set ExternalPostgres back to nil if it hasn't been configured.